recipe/setupext.patch

```diff
@@ -1,16 +1,15 @@
-diff --git setupext.py setupext.py
-index 0ae553a..66c6e02 100755
---- setupext.py
-+++ setupext.py
-@@ -1425,6 +1425,7 @@ class BackendTkAgg(OptionalBackendPackage):
-             tk.destroy()
- 
-         # Save directories and version string to cache
-+#       tcl_lib_dir = tk_lib_dir = r'C:\Tcl%d' % (8 * tuple.__itemsize__)
-         self.tcl_tk_cache = tcl_lib_dir, tk_lib_dir, str(Tkinter.TkVersion)[:3]
-         return self.tcl_tk_cache
- 
-@@ -1531,13 +1532,10 @@ class BackendTkAgg(OptionalBackendPackage):
+--- setupext.py	2016-01-10 19:20:20.000000000 -0300
++++ setupext.py	2016-05-14 16:29:26.477374144 -0300
+@@ -990,7 +990,7 @@
+             ]
+         ext = make_extension('matplotlib._png', sources)
+         pkg_config.setup_extension(
+-            ext, 'libpng', default_libraries=['png', 'z'],
++            ext, 'libpng', default_libraries=['libpng', 'z'],
```

Comment:
This would indicate that it should use a file called "liblibpng.so" would be linked if `pkg-config` fails.  I don't think that's right.